packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { IWizardStyles } from './Wizard.types';
+
+export const getWizardStyles = (): IWizardStyles => {
+  return {
+    wizardContentNavContainer: {
+      height: '601px'
+    },
+    subwayNavSection: {
+      height: '100%',
+      width: '221px',
+      float: 'left'

```

Comment:
We should be using flex here instead of floats. Floats will not respond properly when in RTL layouts and we can also span the container without having to use 100% height with Flex.

Reply:
Fixed